target/linux/octeon/config-5.15

```diff
@@ -44,7 +44,6 @@ CONFIG_CPU_SUPPORTS_HIGHMEM=y
 CONFIG_CPU_SUPPORTS_HUGEPAGES=y
 CONFIG_CRAMFS=y
 CONFIG_CRC16=y
-CONFIG_CRYPTO_BLAKE2S=y
```

Comment:
Did you intend to remove this line?

Reply:
No, but it is removed automatically when running `make kernel_menuconfig` . Should I add whatever pre-req it wants instead?

Reply:
Actually, it seems this option is removed from 5.15 and newer:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/crypto/Kconfig?h=v5.15.120&id=3dd33a09f5dc12ccb0902923c4c784eb0f8c7554

Reply:
I'm not a maintainer so I'm not exactly sure if they'd want ```make kernel_menuconfig``` config changes separated or not

Reply:
I filed https://github.com/openwrt/openwrt/pull/13071 in case the project wants this both separate and fixed for all targets.